packages/dom/src/dom/is-number-input.js

```diff
@@ -2,12 +2,16 @@
  * Check whether the given element is an input field of type number
  * and has a valueAsNumber
  *
- * @param {HTMLElement} element The HTML element.
+ * @param {Element} element The HTML element.
```

Comment:
This should probably be whatever `isHTMLInputElement` accepts.
```suggestion
 * @param {Node} element The HTML element.
```